src/auth/downscopedclient.ts

```diff
@@ -214,10 +214,15 @@ export class DownscopedClient extends AuthClient {
       this.credentialAccessBoundary
     );
 
+    const sourceCredExpireDate =
```

Comment:
Let's add a comment here: 

The STS endpoint will only return the expiration time for the downscoped access token if the
original access token represents a service account. The downscoped token's expiration time will always match the source credential expiration. When no expires_in is returned, we can copy the source credential's expiration time.